### PR TITLE
Fix enwiki dataset

### PR DESCRIPTION
**Pull request by dskhudia**
bert models expect data to contain `input_ids`, `token_type_ids`, `attention_mask` and `labels`. Fix loaded data to use these names.

### CODE REVIEW

File: streaming/text/enwiki.py [hunk 1]
```diff
@@ -68,7 +71,18 @@ def __getitem__(self, idx: int) -> Any:
...
+        if 'input_mask' in obj.keys():
+            obj['attention_mask'] = obj.pop('input_mask')
+        obj.pop('next_sentence_labels')
+        obj.pop('masked_lm_weights')
         for key, value in obj.items():
             dtype = self.field_dtypes[key]
             obj[key] = np.frombuffer(value, dtype)
+        input_len = len(obj['input_ids'])
+        obj['labels'] = np.full((input_len,), -100)
+        obj['labels'][obj['masked_lm_positions']] =  obj['masked_lm_ids']
```
Comment (knighton) [inline streaming/text/enwiki.py:85]: double space

File: streaming/text/enwiki.py [hunk 2]
```diff
@@ -68,7 +71,18 @@ def __getitem__(self, idx: int) -> Any:
...
+        obj.pop('masked_lm_positions')
+        obj.pop('masked_lm_ids')
```
Comment (knighton) [inline streaming/text/enwiki.py:87]: Would it be simpler to return a new dict over changing in-place?

**Review by knighton — APPROVED** [review]

File: streaming/text/enwiki.py (hunk 2 above)
Comment (dskhudia) [reply streaming/text/enwiki.py:87]: What do you prefer? I can do either.

Comment (dskhudia) [reply streaming/text/enwiki.py]: Created a new dict.
